dev/MRTCore/mrt/Core/src/packages.config

```diff
@@ -3,5 +3,5 @@
   <package id="Microsoft.Build.Tasks.Git" version="1.1.0-beta-20204-02" targetFramework="native" developmentDependency="true" />
   <package id="Microsoft.SourceLink.Common" version="1.1.0-beta-21055-01" targetFramework="native" developmentDependency="true" />
   <package id="Microsoft.SourceLink.GitHub" version="1.1.0-beta-20204-02" targetFramework="native" developmentDependency="true" />
-  <package id="Microsoft.Windows.ImplementationLibrary" version="1.0.200519.2" targetFramework="native" />
+  <package id="Microsoft.Windows.ImplementationLibrary" version="1.0.210803.1" targetFramework="native" />
```

Comment:
not required.

Reply:
The MRM project needs it. It might pick it up transitively though since it depends on mrmmin, which depends on Insights, which references wil. Is that what you meant? I can give that a try.

Reply:
I gave the transitive thing a try. I uninstalled the WIL package from mrmmin. I then rebuilt it. It failed with many of these:
```
Error	C1083	Cannot open include file: 'wil\resource.h': No such file or directory	mrmmin	S:\ProjectReunion\dev\MRTCore\mrt\mrm\include\mrm\common\Base.h	15	
```